docs/source/quickstart.rst

```diff
@@ -3,6 +3,219 @@
+     - ``mlflow.log_metric``
+   * - Artifacts
+     - Files produced by the run (for instance, model weights)
+     - ``mlflow.log_artifacts``
+
+This example demonstrates the use of these functions:
+
+.. code-section::
+    .. code-block:: python
+
+        import os
+        from random import random, randint
+        from mlflow import log_metric, log_param, log_artifacts
+
+        if __name__ == "__main__":
+            # Log a parameter (key-value pair)
+            log_param("config_value", randint(0, 100))
+
+            # Log a metric; metrics can be updated throughout the run
+            log_metric("accuracy", random() / 2.0)
+            log_metric("accuracy", random() + 0.1)
+            log_metric("accuracy", random() + 0.2)
+
+            # Log an artifact (output file)
+            if not os.path.exists("outputs"):
+                os.makedirs("outputs")
+            with open("outputs/test.txt", "w") as f:
+                f.write("hello world!")
+            log_artifacts("outputs")
+
+How do I view my MLFlow runs and experiments?
+---------------------------------------------
+
+Once you've run your code, you may view the results with MLFlow's tracking UI. To start the UI, run:
+
+.. code-section::
+
+    .. code-block:: shell
+
+        mlflow ui
+
+And then navigate to http://localhost:5000 in your browser. You will see a page similar to:
+
+.. image:: _static/images/quickstart-ui-screenshot.png
+    :width: 800px
+    :align: center
+
+You are in the **Default** experiment, which now contains the tracking data for your run. You can view the parameters, metrics, and artifacts of your run by clicking on the run name. 
+
+How do I share my MLFlow runs and experiments?
+----------------------------------------------
+
+By default, MLFlow stores tracking data and artifacts in a ``./mlruns`` subdirectory of where you ran the code. You can change this behavior by setting the ``MLFLOW_TRACKING_URI`` environment variable to a different location. For instance, you can set it to a shared filesystem, a SQLAlchemy-compatible database, a tracking server, or a Databricks workspace.
+
+You can run a tracking server on a network-accessible server by running:
+
+.. code-section::
+
+    .. code-block:: shell
+
+        mlflow server
+
+On your development machine, set the ``MLFLOW_TRACKING_URI`` environment variable to the URL of that server. For example, if your tracking server is listening on the default port of **5000** on the machine with IP address **192.168.0.1**, you can set the ``MLFLOW_TRACKING_URI`` environment variable to:
+
+.. code-section::
+
+    .. code-block:: shell
+
+        export MLFLOW_TRACKING_URI=http://192.168.0.1:5000
+
+Now, when you run your code, it will send tracking data to the tracking server. To view the results, run the tracking UI on your local machine and point it to the tracking server:
+
+.. code-section::
+
+    .. code-block:: shell
+
+        mlflow ui --backend-store-uri http://192.168.0.1:5000
```

Comment:
Should just be able to navigate to `http://192.168.0.1:5000` to view the UI

Reply:
Removed this code and changed text to "browse to tracking server"